JetStreamDriver.js

```diff
@@ -1062,11 +1062,15 @@ class Benchmark {
 
     updateUIBeforeRun() {
         if (!dumpJSONResults)
-            console.log(`Running ${this.name}:`);
+            this.updateUIBeforeRunInShell();
         if (isInBrowser)
             this.updateUIBeforeRunInBrowser();
     }
 
+    updateUIBeforeRunInShell() {
```

Comment:
naming nit: This is also not specific to shells, it also prints to the console in the browser. Maybe `updateConsoleBeforeRun`?